### ISSUE TITLE
Dev NANDs freeze on boot

### ISSUE THREAD

**Issue by mariomadproductions** [Arisotura/melonDS#1344]
Any dev NAND dump will freeze on boot. They work in no$gba (although in no$gba the touch screen stops responding part way through the first-use setup, but it seems to work fine if its already set up).

**Comment by Arisotura** [comment]
I think I know why, I recall these reading a certain hardware register to check whether they're running on a dev DSi, and melonDS has it hardcoded to indicate a retail unit

**Comment by mariomadproductions** [comment]
Ah well that sounds like a simple fix if that's true :).

Here's the log anyway: [log.txt](https://github.com/Arisotura/melonDS/files/7858242/log.txt).

I'm using commit f39eeeb.

**Comment by Arisotura** [comment]
oh yeah it seems to be checking for the extra RAM dev units have

**Comment by Arisotura** [comment]
yeah it's also checking 0x04004024 which is the dev/retail type register
